docs/installation.rst

```diff
@@ -324,12 +324,8 @@ While we try not to break master, there are no guarantees on anything. ::
     # get borg from github
     git clone https://github.com/borgbackup/borg.git
 
-    # set python interpreter version e.g. 3.6.0
-    pyenv install 3.6.0
-    pyenv global 3.6.0
-    pyenv local 3.6.0
-
-    virtualenv --python=${pyenv which python} borg-env
+    # create a virtual environment
+    virtualenv --python=${which python3} borg-env
```

Comment:
for more modern python3, one should use python3 -m venv

Reply:
@RonnyPfannschmidt ok, but guess this is something for a global docs / scripts review and then change it everywhere, in a separate PR.